src/components/aside/Aside.jsx

```diff
@@ -1,45 +1,55 @@
 import React from "react";
+import { NavLink } from "react-router-dom";
 import {
   HistoryIcon,
   HomeIcon,
   LikedIcon,
+  PlaylistIcon,
   TrendIcon,
   WatchLaterIcon,
 } from "../../icons/Icons";
 import "./Aside.css";
-export default function Aside() {
+export function Aside() {
   return (
     <aside className="aside">
-      <h2>
-        <span className="aside-icons" >
-          <HomeIcon />
-        </span>
-        Home
-      </h2>
-      <h2>
-        <span className="aside-icons">
-          <TrendIcon />
-        </span>
-        Trend
-      </h2>
-      <h2>
-        <span className="aside-icons">
-          <HistoryIcon />
-        </span>
-        History
-      </h2>
-      <h2>
-        <span className="aside-icons">
-          <WatchLaterIcon />
-        </span>
-        Watch later
-      </h2>
-      <h2>
-        <span className="aside-icons">
-          <LikedIcon />
-        </span>
-        Liked videos
-      </h2>
+      <div className="sidebar">
+        <NavLink to="/" className="btn-link">
+          <span className="aside-icons">
+            <HomeIcon />
+          </span>
+          Home
+        </NavLink>
+        <NavLink to="/trend" className="btn-link">
+          <span className="aside-icons">
+            <TrendIcon />
+          </span>
+          Trend
+        </NavLink>
+        <NavLink to="/history" className="btn-link">
+          <span className="aside-icons">
+            <HistoryIcon />
+          </span>
+          History
+        </NavLink>
+        <NavLink to="/watchlater" className="btn-link">
+          <span className="aside-icons">
+            <WatchLaterIcon />
+          </span>
+          Watch later
+        </NavLink>
+        <NavLink to="/likedvideos" className="btn-link">
+          <span className="aside-icons">
+            <LikedIcon />
+          </span>
+          Liked videos
+        </NavLink>
+        <NavLink to="/playlist" className="btn-link">
+          <span className="aside-icons">
+            <PlaylistIcon />
```

Comment:
Code is repeating here. See if you can use map and make it look clean

Reply:
sure, will do it

Reply:
thanks for point out, i made the changes